src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -621,11 +622,15 @@ private static MethodHandle generateTypeSwitch(MethodHandles.Lookup caller, Clas
         List<EnumDesc<?>> enumDescs = new ArrayList<>();
         List<Class<?>> extraClassLabels = new ArrayList<>();
 
-        byte[] classBytes = ClassFile.of().build(ClassDesc.of(typeSwitchClassName(caller.lookupClass())),
+        byte[] classBytes = ClassFile.of().build(ReferenceClassDescImpl.ofValidatedBinaryName(typeSwitchClassName(caller.lookupClass())),
                 clb -> {
                     clb.withFlags(AccessFlag.FINAL, AccessFlag.SUPER, AccessFlag.SYNTHETIC)
                        .withMethodBody("typeSwitch",
-                                       TYPES_SWITCH_DESCRIPTOR,
+                                       MethodTypeDescImpl.ofValidated(ConstantDescs.CD_int,
+                                            ClassDesc.ofDescriptor(selectorType.descriptorString()),
```

Comment:
`selectorType` can be primitive, therefore verification errors arise at `aload` in the generated code. This is why tests fail on GitHub actions.

Reply:
Dang. I had focused on making the `java/lang/runtime` tests pass locally, thinking that'd be sufficient. Luckily bf68b1d had only a minor benefit. Backing it out and tidying up a bit gives me these numbers:
```
Wall clock:    65.0 ms/op
.taskclock:    75.0 ms/op
.cycles:       255323158 average cycles elapsed
.instructions: 626086160 average instructions retired
```